basic/minimal-e-commerce/init.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+scrypto build && \
+resim reset
+
+export owner_account=$(resim new-account)
+export owner=$(echo $owner_account | grep "Account component address: [a-z0-9]{54}" -oE | cut -d: -f2 | xargs)
+export owner_private_key=$(echo $owner_account | grep "Private key: [a-z0-9]{54}" -oE | cut -d: -f2 | xargs)
+
+export package=$(resim publish target/wasm32-unknown-unknown/release/minimal_e_commerce.wasm | grep "New Package: [a-z0-9]{54}" -oE | cut -d: -f2 | xargs)
+export xrd=$(resim show $owner | grep XRD | cut -d: -f3 | cut -d, -f1 | xargs)
+
+cat <<EOT > manifests/create_catalog.txm
```

Comment:
Minor change: this currently fails if the `manifest` directory is not created then everything else fails. Can you please add in something at the beginning of this script that creates this directory if it doesn't already exist?

Reply:
Sure, `manifests` folder should now be automatically created yet ignored from git. :shipit: